### PR TITLE
Reduce contention on the private blkseq.

### PR DESCRIPTION
**Pull request by riverszhang89**
Increase PRIVATE_BLKSEQ_STRIPES to 8 (same as dtastripe). Also fix the crashes when PRIVATE_BLKSEQ_STRIPES is greater than 1.

### CODE REVIEW

File: bdb/bdb_int.h [hunk 1]
```diff
@@ -1019,7 +1019,8 @@ struct bdb_state_tag {
     DB **blkseq[2];
     time_t blkseq_last_roll_time;
     DB_LSN *blkseq_last_lsn[2];
-    LISTC_T(struct seen_blkseq) blkseq_log_list[2];
+    listc_t *blkseq_log_list;
+    int pvt_blkseq_stripes;
```
Comment (nirbhayc) [inline bdb/bdb_int.h]: Why would you need this new variable?

Comment (riverszhang89) [reply bdb/bdb_int.h]: `bdb_state->pvt_blkseq_stripes` is just a copy from `bdb_state->attr->private_blkseq_stripes`. It is faster than accessing `bdb_state->attr`. It also make sure that we don't end up accessing invalid memory if  `bdb_state->attr->private_blkseq_stripes` is increased at runtime.

**Comment by riverszhang89** [comment]
Roboperf reports significant improvement on sysbench/oltp_insert.

**Comment by adizaimi** [comment]
Nice job Rivers.
Looks good to me.

**Comment by riverszhang89** [comment]
@adizaimi & @nirbhayc Thank you for the reviews!

**Pull request merged**